src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -502,47 +502,47 @@ private static void mapRentalPreferences(
     builder
       .withDropoffCost(
         c
-          .of("bikeRentalDropoffCost")
```

Comment:
If we change the config, I think we should group all of these params into an object so we don't have so many `vehicleRentalXXX` params.

What I mean is this:

```
{
   "vehicleRental": {
       "dropOffCost": 100,
      "dropOffTime": 180
   }
}
```

Reply:
Since we already changing the config, I think we should implement this as well.

Reply:
We discussed in in today's meeting and agreed that it's the way forward.